### DISCUSSION TITLE
Currency "tiles"?

### DISCUSSION THREAD

**Discussion by soxofaan** [Open-EO/openeo-api#421] [General]
https://github.com/Open-EO/openeo-api/blob/f303d65a3291d4cd74dacc0e796803bb5d6fa03b/openapi.yaml#L710-L713

what is "tiles" as a proprietary currency? Or is this a typo?

**Comment by m-mohr** [comment]
That was inspired by SentinelHub. In the past, they billed by tiles (I think that was a 256x256px tile or so).

Maybe we should put currency into quotation marks?!

**Comment by soxofaan** [comment]
If I understand correctly, that just falls under "back-end specific credits", right?

**Reply by m-mohr** [reply]
I've changed 

> or a proprietary currency, e.g. tiles or back-end specific credits.

to

> or a back-end specific unit that is used for billing such as credits, tiles or CPU hours.

See 9c57dfa1

**Comment by soxofaan** [comment]
ok, so discussions can't be closed (like issues) :(

**Reply by m-mohr** [reply]
No, I can only delete it :-(

**Reply by soxofaan** [reply]
better stick to regular issues then
